src/styles/blocks/catalog.scss

```
@@ -11,15 +13,21 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
-
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms;
```

Alisa just ask you to all color property to transition
```suggestion
    transition: color 300ms, transform 300ms;
```

and better to move transition-duration to variable